providers.config.ts

```diff
@@ -222,6 +234,13 @@ exports.providers = {
         lpToken: "0x0000000000000000000000000000000000000000",
         rewardTokens: ["0x73a210637f6F6B7005512677Ba6B3C96bb4AA44B"],
       },
+      "celo-curve-tripool": {
+        pool: "0x32fd7e563c6521ab4d59ce3277bcfbe3317cfd63",
+        gauge: "0x18c45c10a0f41bc3ed8d6324c687335179a40b28",
+        poolType: 3,
+        gaugeMinter: "0xabC000d88f23Bb45525E447528DBF656A9D55bf5",
+        rewardTokens: ["0x173fd7434B8B50dF08e3298f173487ebDB35FD14"],
```

Comment:
this is the CRV token. can we also add it to the list of tokens on celo, pls? thanks